src/app/map/mapbox/mapbox.component.html

```diff
@@ -1 +1 @@
-<div id="map"></div>
\ No newline at end of file
+<div #map id="map"></div>
```

Comment:
Should pull the ID attribute off of this element, just so there are no conflicts if we want multiple map components at some point

Reply:
Good call! updated